### ISSUE TITLE
2025-03-20, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#115]
# Today's Arxiv Papers

(1) [SWEET-RL: Training Multi-Turn LLM Agents on Collaborative Reasoning Tasks](http://arxiv.org/abs/2503.15478v1), 2025-03-19

(2) [Cube: A Roblox View of 3D Intelligence](http://arxiv.org/abs/2503.15475v1), 2025-03-19

(3) [From 1,000,000 Users to Every User: Scaling Up Personalized Preference for User-level Alignment](http://arxiv.org/abs/2503.15463v1), 2025-03-19

(4) [Evaluating Bias in Retrieval-Augmented Medical Question-Answering Systems](http://arxiv.org/abs/2503.15454v1), 2025-03-19

(5) [SkyLadder: Better and Faster Pretraining via Context Window Scheduling](http://arxiv.org/abs/2503.15450v1), 2025-03-19

(6) [Visual Position Prompt for MLLM based Visual Grounding](http://arxiv.org/abs/2503.15426v1), 2025-03-19

(7) [Probing the topology of the space of tokens with structured prompts](http://arxiv.org/abs/2503.15421v1), 2025-03-19

(8) [Real-world validation of a multimodal LLM-powered pipeline for High-Accuracy Clinical Trial Patient Matching leveraging EHR data](http://arxiv.org/abs/2503.15374v1), 2025-03-19

(9) [EfficientLLaVA:Generalizable Auto-Pruning for Large Vision-language Models](http://arxiv.org/abs/2503.15369v1), 2025-03-19

(10) [SemEval-2025 Task 1: AdMIRe -- Advancing Multimodal Idiomaticity Representation](http://arxiv.org/abs/2503.15358v1), 2025-03-19

(11) [SPILL: Domain-Adaptive Intent Clustering based on Selection and Pooling with Large Language Models](http://arxiv.org/abs/2503.15351v1), 2025-03-19

(12) [TruthLens:A Training-Free Paradigm for DeepFake Detection](http://arxiv.org/abs/2503.15342v1), 2025-03-19

(13) [Uncertainty-Guided Chain-of-Thought for Code Generation with LLMs](http://arxiv.org/abs/2503.15341v1), 2025-03-19

(14) [Solla: Towards a Speech-Oriented LLM That Hears Acoustic Context](http://arxiv.org/abs/2503.15338v1), 2025-03-19

(15) [aiXcoder-7B-v2: Training LLMs to Fully Utilize the Long Context in Repository-level Code Completion](http://arxiv.org/abs/2503.15301v1), 2025-03-19

(16) [Inside-Out: Hidden Factual Knowledge in LLMs](http://arxiv.org/abs/2503.15299v1), 2025-03-19

(17) [TROVE: A Challenge for Fine-Grained Text Provenance via Source Sentence Tracing and Relationship Classification](http://arxiv.org/abs/2503.15289v1), 2025-03-19

(18) [SENAI: Towards Software Engineering Native Generative Artificial Intelligence](http://arxiv.org/abs/2503.15282v1), 2025-03-19

(19) [MAMM-Refine: A Recipe for Improving Faithfulness in Generation with Multi-Agent Collaboration](http://arxiv.org/abs/2503.15272v1), 2025-03-19

(20) [Do Chains-of-Thoughts of Large Language Models Suffer from Hallucinations, Cognitive Biases, or Phobias in Bayesian Reasoning?](http://arxiv.org/abs/2503.15268v1), 2025-03-19

(21) [LEGION: Learning to Ground and Explain for Synthetic Image Detection](http://arxiv.org/abs/2503.15264v1), 2025-03-19

(22) [Efficient allocation of image recognition and LLM tasks on multi-GPU system](http://arxiv.org/abs/2503.15252v1), 2025-03-19

(23) [Automated Non-Functional Requirements Generation in Software Engineering with Large Language Models: A Comparative Study](http://arxiv.org/abs/2503.15248v1), 2025-03-19

(24) [BigO(Bench) -- Can LLMs Generate Code with Controlled Time and Space Complexity?](http://arxiv.org/abs/2503.15242v1), 2025-03-19

(25) [Exploring Large Language Models for Word Games:Who is the Spy?](http://arxiv.org/abs/2503.15235v1), 2025-03-19

(26) [When LLMs Meet API Documentation: Can Retrieval Augmentation Aid Code Generation Just as It Helps Developers?](http://arxiv.org/abs/2503.15231v1), 2025-03-19

(27) [Benchmarking Large Language Models for Handwritten Text Recognition](http://arxiv.org/abs/2503.15195v1), 2025-03-19

(28) [Optimizing Retrieval Strategies for Financial Question Answering Documents in Retrieval-Augmented Generation Systems](http://arxiv.org/abs/2503.15191v1), 2025-03-19

(29) [A Review on Large Language Models for Visual Analytics](http://arxiv.org/abs/2503.15176v1), 2025-03-19

(30) [Comparing Llama3 and DeepSeekR1 on Biomedical Text Classification Tasks](http://arxiv.org/abs/2503.15169v1), 2025-03-19

(31) [Aligning Crowd-sourced Human Feedback for Reinforcement Learning on Code Generation by Large Language Models](http://dx.doi.org/10.1109/TBDATA.2024.3524104), 2025-03-19

(32) [Increasing the Robustness of the Fine-tuned Multilingual Machine-Generated Text Detectors](http://arxiv.org/abs/2503.15128v1), 2025-03-19

(33) [Text-Derived Relational Graph-Enhanced Network for Skeleton-Based Action Segmentation](http://arxiv.org/abs/2503.15126v1), 2025-03-19

(34) [Exploring Model Editing for LLM-based Aspect-Based Sentiment Classification](http://arxiv.org/abs/2503.15117v1), 2025-03-19

(35) [Reasoning Effort and Problem Complexity: A Scaling Analysis in LLMs](http://arxiv.org/abs/2503.15113v1), 2025-03-19

(36) [OpenLLM-RTL: Open Dataset and Benchmark for LLM-Aided Design RTL Generation](http://arxiv.org/abs/2503.15112v1), 2025-03-19

(37) [VIPER: Visual Perception and Explainable Reasoning for Sequential Decision-Making](http://arxiv.org/abs/2503.15108v1), 2025-03-19

(38) [Towards Understanding the Safety Boundaries of DeepSeek Models: Evaluation and Findings](http://arxiv.org/abs/2503.15092v1), 2025-03-19

(39) [Intelligent Spatial Perception by Building Hierarchical 3D Scene Graphs for Indoor Scenarios with the Help of LLMs](http://arxiv.org/abs/2503.15091v1), 2025-03-19

(40) [LogiAgent: Automated Logical Testing for REST Systems with LLM-Based Multi-Agents](http://arxiv.org/abs/2503.15079v1), 2025-03-19

(41) [ELTEX: A Framework for Domain-Driven Synthetic Data Generation](http://arxiv.org/abs/2503.15055v1), 2025-03-19

(42) [Studying and Understanding the Effectiveness and Failures of Conversational LLM-Based Repair](http://arxiv.org/abs/2503.15050v1), 2025-03-19

(43) [SPADE: Systematic Prompt Framework for Automated Dialogue Expansion in Machine-Generated Text Detection](http://arxiv.org/abs/2503.15044v1), 2025-03-19

(44) [Learning 4D Panoptic Scene Graph Generation from Rich 2D Visual Scene](http://arxiv.org/abs/2503.15019v1), 2025-03-19

(45) [LLM Alignment for the Arabs: A Homogenous Culture or Diverse Ones?](http://arxiv.org/abs/2503.15003v1), 2025-03-19

(46) [Right Answer, Wrong Score: Uncovering the Inconsistencies of LLM Evaluation in Multiple-Choice Question Answering](http://arxiv.org/abs/2503.14996v1), 2025-03-19

(47) [ML-Triton, A Multi-Level Compilation and Language Extension to Triton GPU Programming](http://arxiv.org/abs/2503.14985v1), 2025-03-19

(48) [Neuro Symbolic Knowledge Reasoning for Procedural Video Question Answering](http://arxiv.org/abs/2503.14957v1), 2025-03-19

(49) [ChatStitch: Visualizing Through Structures via Surround-View Unsupervised Deep Image Stitching with Collaborative LLM-Agents](http://arxiv.org/abs/2503.14948v1), 2025-03-19

(50) [UPME: An Unsupervised Peer Review Framework for Multimodal Large Language Model Evaluation](http://arxiv.org/abs/2503.14941v1), 2025-03-19

(51) [VisNumBench: Evaluating Number Sense of Multimodal Large Language Models](http://arxiv.org/abs/2503.14939v1), 2025-03-19

(52) [Proceedings of the 3rd Italian Conference on Big Data and Data Science (ITADATA2024)](http://arxiv.org/abs/2503.14937v1), 2025-03-19

(53) [Enhancing Code LLM Training with Programmer Attention](http://arxiv.org/abs/2503.14936v1), 2025-03-19

(54) [FAVOR-Bench: A Comprehensive Benchmark for Fine-Grained Video Motion Understanding](http://arxiv.org/abs/2503.14935v1), 2025-03-19

(55) [Prada: Black-Box LLM Adaptation with Private Data on Resource-Constrained Devices](http://arxiv.org/abs/2503.14932v1), 2025-03-19

(56) [UTFix: Change Aware Unit Test Repairing using LLM](http://arxiv.org/abs/2503.14924v1), 2025-03-19

(57) [GenM$^3$: Generative Pretrained Multi-path Motion Model for Text Conditional Human Motion Generation](http://arxiv.org/abs/2503.14919v1), 2025-03-19

(58) [MASS: Mathematical Data Selection via Skill Graphs for Pretraining Large Language Models](http://arxiv.org/abs/2503.14917v1), 2025-03-19

(59) [POSTA: A Go-to Framework for Customized Artistic Poster Generation](http://arxiv.org/abs/2503.14908v1), 2025-03-19

(60) [Deep Contrastive Unlearning for Language Models](http://arxiv.org/abs/2503.14900v1), 2025-03-19

(61) [Mitigating Object Hallucinations in MLLMs via Multi-Frequency Perturbations](http://arxiv.org/abs/2503.14895v1), 2025-03-19

(62) [MetaLadder: Ascending Mathematical Solution Quality via Analogical-Problem Reasoning Transfer](http://arxiv.org/abs/2503.14891v1), 2025-03-19

(63) [Pseudo-Relevance Feedback Can Improve Zero-Shot LLM-Based Dense Retrieval](http://arxiv.org/abs/2503.14887v1), 2025-03-19

(64) [Envisioning an AI-Enhanced Mental Health Ecosystem](http://arxiv.org/abs/2503.14883v1), 2025-03-19

(65) [Communication-Efficient Distributed On-Device LLM Inference Over Wireless Networks](http://arxiv.org/abs/2503.14882v1), 2025-03-19

(66) [Unlocking the Capabilities of Vision-Language Models for Generalizable and Explainable Deepfake Detection](http://arxiv.org/abs/2503.14853v1), 2025-03-19

(67) [LogLLaMA: Transformer-based log anomaly detection with LLaMA](http://arxiv.org/abs/2503.14849v1), 2025-03-19

(68) [Think Like Human Developers: Harnessing Community Knowledge for Structured Code Reasoning](http://arxiv.org/abs/2503.14838v1), 2025-03-19

(69) [Robust Transmission of Punctured Text with Large Language Model-based Recovery](http://arxiv.org/abs/2503.14831v1), 2025-03-19

(70) [Fake Runs, Real Fixes -- Analyzing xPU Performance Through Simulation](http://arxiv.org/abs/2503.14781v1), 2025-03-18

(71) [Uncertainty Distillation: Teaching Language Models to Express Semantic Confidence](http://arxiv.org/abs/2503.14749v1), 2025-03-18

(72) [CodingGenie: A Proactive LLM-Powered Programming Assistant](http://arxiv.org/abs/2503.14724v1), 2025-03-18

(73) [TestForge: Feedback-Driven, Agentic Test Suite Generation](http://arxiv.org/abs/2503.14713v1), 2025-03-18

(74) [Generating Medically-Informed Explanations for Depression Detection using LLMs](http://arxiv.org/abs/2503.14671v1), 2025-03-18

(75) [ConQuer: A Framework for Concept-Based Quiz Generation](http://arxiv.org/abs/2503.14662v1), 2025-03-18

(76) [RAGO: Systematic Performance Optimization for Retrieval-Augmented Generation Serving](http://arxiv.org/abs/2503.14649v1), 2025-03-18

(77) [Towards More Economical Context-Augmented LLM Generation by Reusing Stored KV Cache](http://arxiv.org/abs/2503.14647v1), 2025-03-18

(78) [Assessing Large Language Models for Automated Feedback Generation in Learning Programming Problem Solving](http://arxiv.org/abs/2503.14630v1), 2025-03-18

(79) [Retrieval-Augmented Simulacra: Generative Agents for Up-to-date and Knowledge-Adaptive Simulations](http://arxiv.org/abs/2503.14620v1), 2025-03-18

(80) [Image Captioning Evaluation in the Age of Multimodal LLMs: Challenges and Future Perspectives](http://arxiv.org/abs/2503.14604v1), 2025-03-18

(81) [Command R7B Arabic: A Small, Enterprise Focused, Multilingual, and Culturally Aware Arabic LLM](http://arxiv.org/abs/2503.14603v1), 2025-03-18

(82) [MusicInfuser: Making Video Diffusion Listen and Dance](http://arxiv.org/abs/2503.14505v1), 2025-03-18

(83) [Aligning Multimodal LLM with Human Preference: A Survey](http://arxiv.org/abs/2503.14504v1), 2025-03-18

(84) [Temporal Consistency for LLM Reasoning Process Error Identification](http://arxiv.org/abs/2503.14495v1), 2025-03-18

(85) [Engineering Scientific Assistants using Interactive Structured Induction of Programs](http://arxiv.org/abs/2503.14488v1), 2025-03-18

(86) [Gricean Norms as a Basis for Effective Collaboration](http://arxiv.org/abs/2503.14484v1), 2025-03-18

(87) [Creation-MMBench: Assessing Context-Aware Creative Intelligence in MLLM](http://arxiv.org/abs/2503.14478v2), 2025-03-18

(88) [Calibrating Verbal Uncertainty as a Linear Feature to Reduce Hallucinations](http://arxiv.org/abs/2503.14477v1), 2025-03-18

(89) [DAPO: An Open-Source LLM Reinforcement Learning System at Scale](http://arxiv.org/abs/2503.14476v1), 2025-03-18

(90) [EnvBench: A Benchmark for Automated Environment Setup](http://arxiv.org/abs/2503.14443v1), 2025-03-18

(91) [LLM-FE: Automated Feature Engineering for Tabular Data with LLMs as Evolutionary Optimizers](http://arxiv.org/abs/2503.14434v1), 2025-03-18

(92) [PLAY2PROMPT: Zero-shot Tool Instruction Optimization for LLM Agents via Tool Play](http://arxiv.org/abs/2503.14432v1), 2025-03-18

(93) [Unifying Text Semantics and Graph Structures for Temporal Text-attributed Graphs with Large Language Models](http://arxiv.org/abs/2503.14411v1), 2025-03-18

(94) [Large Language Models for Virtual Human Gesture Selection](http://arxiv.org/abs/2503.14408v1), 2025-03-18

(95) [From "Hallucination" to "Suture": Insights from Language Philosophy to Enhance Large Language Models](http://arxiv.org/abs/2503.14392v1), 2025-03-18

(96) [How much do LLMs learn from negative examples?](http://arxiv.org/abs/2503.14391v1), 2025-03-18

(97) [Good/Evil Reputation Judgment of Celebrities by LLMs via Retrieval Augmented Generation](http://arxiv.org/abs/2503.14382v1), 2025-03-18

(98) [On the Standard Performance Criteria for Applied Control Design: PID, MPC or Machine Learning Controller?](http://arxiv.org/abs/2503.14379v1), 2025-03-18

(99) [Impossible Videos](http://arxiv.org/abs/2503.14378v1), 2025-03-18

(100) [MANTRA: Enhancing Automated Method-Level Refactoring with Contextual RAG and Multi-Agent LLM Collaboration](http://arxiv.org/abs/2503.14340v1), 2025-03-18
